src/main/java/com/android/volley/toolbox/DiskBasedAsyncCache.java

```diff
@@ -76,12 +83,22 @@ public void completed(Integer result, Void v) {
                                 return;
                             }
                             byte[] data = buffer.array();
+                            try {
+                                afc.close();
+                            } catch (IOException e) {
+                                VolleyLog.e(e, "Failed to close channel");
```

Comment:
super small nit: having more descriptive error logs can be helpful to narrow down where the error comes from. This is why I'd suggest to say something like `Failed to close channel after successful read` here, and `Failed to close channel after failed read attempt` below?